resources/qml/WelcomePages/AddUltimakerOrThirdPartyPrinter.qml

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2022 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+
+import UM 1.5 as UM
+import Cura 1.1 as Cura
+
+
+//
+// This component contains the content for the "Add a printer" (network) page of the welcome on-boarding process.
+//
```

Comment:
I like this!

Reply:
Thanks!

Comment was already there though, the file moved ;)